### ISSUE TITLE
Prometheus `quantile` variable produces error `t.replace is not a function`

### ISSUE THREAD

**Issue by jakubdal** [grafana/grafana#12348]
### What Grafana version are you using?

Grafana v5.1.3 (087143285)

![quantile_grafanaversion](https://user-images.githubusercontent.com/15217635/41609986-4dba1462-73ec-11e8-9a23-d6452188040b.png)

### What datasource are you using?

Prometheus, version 2.3.0 (branch: HEAD, revision: 290d71791a507a5057b9a099c9d48703d86dc941)

### What OS are you running grafana on?

Debian 9.4

### What did you do?

Added a template variable `quantile` from `quantile` label.

![quantile_variablecreation](https://user-images.githubusercontent.com/15217635/41610218-eb1d7b5e-73ec-11e8-937d-e8f81e590c19.png)

### What was the expected result?

Using `quantile` variable to choose `quantile` to be displayed on graphs.

### What happened instead?
 
An error:

![quantile_error](https://user-images.githubusercontent.com/15217635/41610225-f19fc27a-73ec-11e8-99ee-7ca4ff1edbdf.png)

After switching screens:

![quantile_switch_screens](https://user-images.githubusercontent.com/15217635/41610233-f6289236-73ec-11e8-9a0f-c8c304c6fae9.png)

**Comment by bergquist** [comment]
Strange. I can create a similar dashboard without problems. https://play.grafana.org/d/McdataNmk/prometheus-quantile-variable?orgId=2&var-quantile=0.25 

not sure what differs. Any chance you can compare the dashboards?

**Comment by jakubdal** [comment]
I'm sorry @bergquist , but it seems that your link doesn't work. Could you provide another one please?

**Comment by jakubdal** [comment]
No problem, thank you for trying to help :)

Strange, indeed it seems to be the same case, yet it produces different results. What kind of Prometheus metric is that? In my app I use Summary, maybe that's the case?

**Comment by jakubdal** [comment]
Wait, ok, this is getting weird. I've tried reproducing my own problem and using your example with `go_gc_duration_seconds`. It worked just fine. So then I tried my own function. It seems to (once again) work just fine!

I honestly have no clue about what happened - I did not update neither Grafana nor Prometheus. From what I remember, I have tweaked my Prometheus settings a bit, but I cannot tell you what exactly changed this to work. I do believe this can be closed for now, as even I cannot reproduce the bug?
